### ISSUE TITLE
Question: Does Vesta now split nginx config files instead of adding everything to one config?

### ISSUE THREAD

**Issue by DaniWinter** [serghey-rodin/vesta#1447]
See title

**Comment by noogen** [comment]
I heard it was talked about.  Was the changed already made it to v18?  If this is true then celebration is in order. 👍

**Comment by madeITBelgium** [comment]
Yes, all newly created domains or after a rebuild all domains have their own file. Just deleting the snginx.conf and nginx.conf is not a good idea because old domains are still configured in this files. After a rebuild of the user, the old config files are removed.

**Comment by DaniWinter** [comment]
Great! How do I rebuild a user?

**Comment by robindirksen1** [comment]
Run the command `v-rebuild-user [USER]` - see https://github.com/serghey-rodin/vesta/blob/master/bin/v-rebuild-user

**Comment by rogeriotaques** [comment]
I started getting the same issue yesterday on one of my servers, and actually, it's only happening to one of the 5 domains hosted in there. All the other 4 seems to be ok.

The current setup of that server is: 
- `Ubuntu 18.04(x86_64)`
- `Vesta  0.9.8 (amd64) release: 26`

Yesterday, after `v-update-letsencrypt-ssl` run, I got an email notification like this:

```
nginx: [warn] the "ssl" directive is deprecated, use the "listen ... ssl" directive instead in /home/USER/conf/web/DOMAIN.nginx.ssl.conf:4
```

I manually updated the mentioned files (5 in total) and restarted `nginx`. Everything went back to normal and worked fine ... until this morning when I got another email notification telling me that the server was unreachable.

I noticed `nginx` was off again. When I restarted it from Vesta panel, it worked.

Then, I tried manually run `v-update-letsencrypt-ssl` and `nginx` went off!

After find this thread, I've tried to rebuild the user (as suggested above) which has not raised any issue, however, when I ran `v-update-letsencrypt-ssl` again (testing) I got `nginx` off and the same (first) email notification as before:

```
nginx: [warn] the "ssl" directive is deprecated, use the "listen ... ssl" directive instead in /home/USER/conf/web/DOMAIN.nginx.ssl.conf:4
```

For now, I've suspended the cronjob that triggers `v-update-letsencrypt-ssl`, (once more) manually updated the 5 files that has the `deprecated directive` and restarted Nginx., what seems to be working. But, since the issue was not fixed, I'm wondering if you guys may have any suggestions.
